lib/core/utils/progress_dialog_utils.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+class ProgressDialogUtils {
+  static bool isProgressVisible = false;
```

Comment:
it should be private if you are encapsulating the hidden logic and providing a method for hiding a dialog.

```suggestion
  static bool _isProgressVisible = false;
```

Reply:
progress dialog class has been removed from the code because there is no need of loader. We will use shimmer package in future